userspace/falco/falco.cpp

```diff
@@ -482,6 +485,7 @@ int falco_init(int argc, char **argv)
         {"stats_interval", required_argument, 0},
         {"support", no_argument, 0},
         {"unbuffered", no_argument, 0, 'U'},
+		{"udig", no_argument, 0, 'u'},
```

Comment:
```suggestion
	{"udig", no_argument, 0, 'u'},
```

Reply:
Also, I think being more verbose here would help. I'm referring to naming the variable to something like `userspace` directly.